test/Libraries/RevitIntegrationTests/BugTests.cs

```diff
@@ -763,6 +763,24 @@ public void CanOpenAndRunOtherFilesAfterOpeningFileWithSelectNode()
             RunCurrentModel();
         }
 
+        [Test]
+        [Category("RegressionTests")]
+        [TestModel(@".\empty.rfa")]
+        public void AllElementsInActiveViewReturnsViewableElements()
+        {
+            string filePath = Path.Combine(workingDirectory, @".\Bugs\MAGN_7641_simplified.dyn");
+            string testPath = Path.GetFullPath(filePath);
+
+            ViewModel.OpenCommand.Execute(testPath);
+            AssertNoDummyNodes();
+            RunCurrentModel();
```

Comment:
@sharadkjaiswal 
There is a limitation here!
Right now the test case is running in the background thread. In the process, we have no chance to manipulate the views.